app/matplotlib.ipynb

```diff
@@ -233,22 +233,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, ax = plt.subplots(1, 2, figsize=(12, 6))\n",
+    "\n",
+    "fig.suptitle(\"Subplots\")\n",
+    "\n",
+    "ax[0].plot(x, y1)\n",
+    "ax[0].set_title(\"Sine Function\")\n",
+    "ax[0].set_xlabel(\"X Axis\")\n",
+    "ax[0].set_ylabel(\"Y Axis\")\n",
+    "\n",
+    "ax[1].plot(x, y2)\n",
```

Comment:
Different colors for each plot are not set. Use the `color` parameter in `ax[0].plot()` and `ax[1].plot()` to set different colors for the sine and cosine functions.